publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -114,19 +114,17 @@ private void readSpringProperties() {
                 rabbitMqProperties.setExchangeName(rabbitmqInstanceObject.get("exchangeName").asText());
                 rabbitMqProperties.setCreateExchangeIfNotExisting(rabbitmqInstanceObject.get("createExchangeIfNotExisting").asBoolean());
                 rabbitMqProperties.setDomainId(rabbitmqInstanceObject.get("domainId").asText());
-                if((rabbitmqInstanceObject.get("channelsCount") != null) ) {
+                if ((rabbitmqInstanceObject.get("channelsCount") != null)) {
```

Comment:
I'd store result of `rabbitmqInstanceObject.get("channelsCount")` into a var to avoid next invocation of the same method...

```
channelsCount = rabbitmqInstanceObject.get("channelsCount");
if (channelsCount != null) {
   rabbitMqProperties.setChennelsCount(Integer.parseInt(channelsCount.asText()));
}
```

The same for other properties.

Reply:
Made the necessery changes